cypress/e2e/hub/collections.cy.ts

```diff
@@ -222,9 +222,23 @@ describe('Collections Details View', () => {
 
   it.skip('user can deprecate a collection', () => {});
 
-  it.skip('user can delete version from system', () => {});
+  it('user can delete version from system', () => {
+    cy.approveCollection(collection, namespace, '1.0.0');
+    cy.galaxykit('task wait all');
+    cy.get('[data-cy="table-view"]').click();
+    cy.clickLink(collection);
+    cy.wait(400);
```

Comment:
Lets avoid hardcoded values with `cy.wait`, it might lead to failures.

See: https://filiphric.com/waiting-in-cypress-and-how-to-avoid-it

Reply:
@MilanPospisil and I discussed this- unfortunately, due to some framework glitch that causes reloading we have to keep that in for now.

Reply:
Nevermind, I found a way around it by waiting for the correct URL!